docs/tutorials/configure/scuttlebot.md

```diff
@@ -159,164 +240,185 @@ This will be similar to adding the right motor.
-       "attributes": {
-           "board": "<board_name>",
-           "connection_type": "i2c",
-            "i2c_attributes": {
-                "i2c_bus": "main",
-                "i2c_addr": 64
-            }
-       }
-    }
-  ```
-
-- Right encoder:
-  - Configure the left encoder with *Name* `renc`, **Type** `encoder`, and **Model** `AMS-AS5048`.
-  - Paste the following in the **Attributes** field, changing the board name to match the name of your board:
-
-  ```json
-  {
-      "name": "<encoder_name>",
-      "type": "encoder",
-      "model": "AMS-AS5048"
-      "board": "<board_name>",
-       "attributes": {
-           "board": "<board_name>",
-           "connection_type": "i2c",
-            "i2c_attributes": {
-                "i2c_bus": "main",
-                "i2c_addr": 65
-            }
-       }
-    }
-  ```
+Finally, add a camera to your SCUTTLE Robot.
 
-## Configuring the camera
+{{< tabs name="Configure a Webcam" >}}
+{{% tab name="Config Builder" %}}
 
-Finally, we'll add a camera to your SCUTTLE Robot.
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your camera, select the type `camera`, and select the `webcam` model.
 
-1. Enter a name of your choice in **Name**.
-2. Enter "camera" in **Model**.
-3. Click **Create Component**.
+Click **Create component**.
 
-Now, you'll see the config panel for the camera component:
-![Camera component config panel](/tutorials/scuttlebot/pi-cam-control.png)
+{{< imgproc src="/components/camera/configure-webcam.png" alt="Configuration of a webcam camera in the Viam app config builder." resize="600x" >}}
 
-On the camera config panel, set the `video_path`.
+If you click on the **Video Path** field while your robot is live, a drop down autopopulates with identified camera paths.
 
-Once you save the config, you'll be able to see your camera's stream in the **Control** tab.
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
 
-## On completion
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "name": "Webcam",
+    "type": "camera",
+    "model" : "webcam",
+    "attributes": {
+        "video_path": "video0"
+    }
+}
+```
 
-After successfully completing this tutorial, you have a fully configured SCUTTLE Robot.
-You can drive it around and view its camera stream.
+{{% /tab %}}
+{{< /tabs >}}
 
-To take things to the next level, check out our [Color Detection with SCUTTLE Robot](/tutorials/services/color-detection-scuttle/) tutorial or create your own camera-related tutorial.
-Alternatively, you can  check out our [Bluetooth Gamepad For SCUTTLE](/tutorials/control/scuttle-gamepad/) tutorial or our [Line Follower Robot tutorial](/tutorials/services/webcam-line-follower-robot/).
+## View the camera stream
 
-## Documents referenced
+{{< readfile "/static/include/components/camera-view-camera-stream.md" >}}
```

Comment:
This snippet references point cloud data but that doesn't apply here so maybe better not to use the snippet??